app/views/middleware_server/_add_jdbc_driver.html.haml

```diff
@@ -0,0 +1,136 @@
+            .col-md-8
+              %input.form-control#jdbc_module_name_input{:type           => "text",
+                                                          "name"         => "jdbc_module_name_input",
+                                                          "ng-disabled"  => "!jdbcDriverModel.filePath",
+                                                          "ng-required"  => "true",
+                                                          "ng-minlength" => "6",
+                                                          "ng-maxlength" => "256",
+                                                          "placeholder"  => _("Enter JDBC Module name"),
+                                                          "ng-model"     => "jdbcDriverModel.moduleName"}
+
+            .col-sm-2
+            .col-sm-10
+              %p.ng-error-messages{"ng-show" => "jdbcAddForm.jdbc_module_name_input.$dirty && jdbcAddForm.jdbc_module_name_input.$error.required"}
+                = _("Required")
+              %p.ng-error-messages{"ng-show" => "jdbcAddForm.jdbc_module_name_input.$dirty && jdbcAddForm.jdbc_module_name_input.$error.minlength"}
+                = _("minimum length is 6")
+              %p.ng-error-messages{"ng-show" => "jdbcAddForm.jdbc_module_name_input.$dirty && jdbcAddForm.jdbc_module_name_input.$error.maxlength"}
+                = _("Maximum length is 256")
+
+
+          .form-group
+            %label.col-md-4.control-label{:for => "jdbc_driver_class_input"}
+              = _("Driver Class")
+            .col-md-8
+              %input.form-control#jdbc_driver_class_input{:type          => "text",
+                                                          "name"         => "jdbc_driver_class_input",
+                                                          "ng-disabled"  => "!jdbcDriverModel.filePath",
+                                                          "ng-required"  => "true",
+                                                          "ng-minlength" => "6",
+                                                          "ng-maxlength" => "256",
+                                                          "placeholder"  => _("Enter JDBC Driver Class"),
+                                                          "ng-model"     => "jdbcDriverModel.driverClass"}
+
+            .col-sm-2
+            .col-sm-10
+              %p.ng-error-messages{"ng-show" => "jdbcAddForm.jdbc_driver_class_input.$dirty && jdbcAddForm.jdbc_driver_class_input.$error.required"}
+                = _("Required")
+              %p.ng-error-messages{"ng-show" => "jdbcAddForm.jdbc_driver_class_input.$dirty && jdbcAddForm.jdbc_driver_class_input.$error.minlength"}
+                = _("minimum length is 6")
+              %p.ng-error-messages{"ng-show" => "jdbcAddForm.jdbc_driver_class_input.$dirty && jdbcAddForm.jdbc_driver_class_input.$error.maxlength"}
+                = _("Maximum length is 256")
+
+          .form-group{"ng-if" => false}
```

Comment:
Again, WIP. Not sure we want to present this to the user.. it is supposed to specify the major/minor version of the driver, in case the jar contains multiple versions (AFAICT). I know we can't pick "custom" versions, otherwise it will fail with something like "driver with major version 1 and minor version 9 not found".
@pilhuhn, wdyt? should these fields be visible for the user (they are optional, tho)